akka-http-core/src/main/scala/akka/http/impl/engine/ws/FrameEventRenderer.scala

```diff
@@ -22,21 +22,19 @@ private[http] final class FrameEventRenderer extends GraphStage[FlowShape[FrameE
   val out = Outlet[ByteString]("FrameEventRenderer.out")
   override val shape = FlowShape(in, out)
 
-  override def createLogic(inheritedAttributes: Attributes): GraphStageLogic = new GraphStageLogic(shape) {
-
-    val Initial = new InHandler {
+  override def createLogic(inheritedAttributes: Attributes): GraphStageLogic =
```

Comment:
I think we should leave this one as is because it makes the state machine a bit easier to follow (Initial -> subsequent render states)

Reply:
okay：）